.gitmodules

```diff
@@ -15,7 +15,7 @@
 	path = externals/kokkos
 	url = git@github.com:E3SM-Project/kokkos.git
 	branch = e3sm/kokkos
-[submodule "components/cam/src/physics/rrtmgp/external"]
+[submodule "RRMGP"]
```

Comment:
Just eyeballing, isn't the 'T' missing in RRTMGP here?

Reply:
Or does this not matter in terms of finding submodule....

Reply:
Not for finding since what is in quotes is just a label to use for submodule commands.  But the T should be there.

Reply:
Thanks for noticing the typo. I will fix it.